src/styles/blocks/rate-stars.scss

```diff
@@ -0,0 +1,17 @@
+.rate-stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../../src/images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 15px;
+    height: 15px;
+    margin-right: 4px;
+
+    &:last-of-type {
+      background-image: url(../../src/images/star.svg);
+      margin-right: 0;
+    }
+  }
```

Comment:
You should use styles from Stars task with styling according to modifiers like `stars--4`

You can also apply SCSS loop here.